### ISSUE TITLE
Allow the specification of a minimum heat pump size.

### ISSUE THREAD

**Issue by afontani** [NREL/OpenStudio-HPXML#1275]
Under aggressive envelope upgrades, ResStock (LA100 model) is seeing a lot of unmet hours from heat pumps whose sizes are small ~0.5 ton. We would like to be able to set a minimum capacity. 

I think the `AllowIncreasedFixedCapacities` argument with the user-specified capacity would allow for this functionality.

**Comment by shorowit** [comment]
@afontani I would prefer if we could identify the cause of the undersized heat pump. Can you provide an HPXML or the steps needed to reproduce the issue?

**Comment by ekpresent** [comment]
@noahsandoval let's work together today to get Scott an example file or two.

**Comment by noahsandoval** [comment]
@ekpresent @afontani @shorowit 
[Here are the .xml files ](https://nrel-my.sharepoint.com/:f:/g/personal/nsandova_nrel_gov/ElkNAzoAWB5NhwVDMtf7J7cBteAgiWPECGCd1DOJi1qwFQ?e=zWfvtz)for building_id 45552

[Here are the .xml files](https://nrel-my.sharepoint.com/:f:/g/personal/nsandova_nrel_gov/EpdG7mtEv_JEhnUgbAgZwXABrFdN7hGH4L26xNuhUSkQDw?e=65KgQO) for building_id 21340

Both are SFDs with an unusually high number of unmet cooling hours.

**Comment by shorowit** [comment]
Note: These are some _seriously_ unrealistic buildings. Building 45552 is a 633 sqft SFD home, yet is 2 stories and has 9 occupants. Building 21340 is a 328 sqft SFD home (is that even possible?!), yet is 2 stories and has 5 occupants. Perhaps the process by which these samples are produced should be scrutinized...

That said, I've dug in a bit and much of the unmet hours appears to stem from having a high number of occupants -- much greater than the assumed number of occupants (#bedrooms+1) in our Manual J calculation. This is easy to address: use the assumed number of occupants for asset calculations and use the provided number of occupants for operational calculations. PR coming; it's also an easy fix to backport. In building 45552, the cooling unmet hours are reduced from 5507 to 1646. In building 21340, it's reduced from 5597 to 2121.

Aside from this, I did a quick comparison of the OS-HPXML design loads to another software tool and they match up pretty well other than the infiltration/ventilation load. I will plan to look at that a little closer.

**Comment by afontani** [comment]
@shorowit :Point taken. I looked at our characteristics and although the models were created for the LA100 ES project, it is still a potential problem with `project_national` in ResStock. 

I created an issue https://github.com/NREL/resstock-estimation/issues/351.

**Comment by ekpresent** [comment]
I finally took a look at the full population (results.csv) and I don't think @shorowit's takeaways tell the full story. Yes, some of the most egregious cases involve homes that probably don't make sense geometrically. Yes, this is much worse when the occupant count is above what our sizing algorithm uses. But usage level alone seems to play a big role. Take a look.  (This is the percent of models with an increase in unmet hours of 1,000 hours or more, I could use another threshold - this one is arbitrary.)

![Gains of more than 1000 unmet cooling hours](https://user-images.githubusercontent.com/46542418/216727325-7b825fd4-b20a-41fd-9768-4093bc30ff4a.png)

So I do agree that this is internal gains driven with these very insulated high internal gains homes. But I think there's more than just sizing at play here. I think this may be showing some holes in our natural ventilation. In a highly diurnal climate, a home with high internal gains that is experiencing temperatures well above the setpoint is going to be using techniques other than the heat pump.

**Comment by shorowit** [comment]
Sorry, I had started to type up some additional information but got distracted. Yes, I noticed that the plug loads multiplier, like the number of occupants, is also a major culprit. That's why I ended up reworking my PR into [this](https://github.com/NREL/OpenStudio-HPXML/pull/1282), which can allow resstock to better inform the Manual J sizing calculations for these types of homes.

These are super low-load homes that are completely dominated by internal gains; if the internal gains are high (either because of plug loads or occupants), they can exceed the (generally pretty conservative) defaults we use from Manual J. But we're talking about selecting cooling capacities that are too small by _fractions_ of a ton. Sure it looks scary to have thousands of hours of unmet cooling load, but if you had that extra fraction of a ton of cooling capacity, it wouldn't result in large increases in cooling energy use.

**Comment by shorowit** [comment]
But if you really need to see those unmet hours drop towards zero, I can provide some additional easy-to-implement solutions that should help.

**Comment by noahsandoval** [comment]
Update: I have still not been able to rerun this .yml calling the `add_component_loads`  and `include_timeseries_component_loads` measures due to Eagle being down. I'll keep trying over the weekend and hope we can have something by our meeting on Monday.

**Comment by shorowit** [comment]
> Aside from this, I did a quick comparison of the OS-HPXML design loads to another software tool and they match up pretty well other than the infiltration/ventilation load. I will plan to look at that a little closer.

I pored over our calculations and they look good. I discovered that the other software tool's results were moderately different for infiltration/ventilation because of the difference between the two Los Angeles sites in Manual J. Switched to the AP site and now they show good agreement. (I did end up tweaking our assumption about indoor RH; Manual J recommends 45% - 55% and we were assuming 55%. Updated it to be 50%. Perhaps it could be another input exposed in my PR.)
